numba/npyufunc/parallel.py

```diff
@@ -211,24 +230,25 @@ def build(self, cres):
 
         return dtypenums, ptr, env
 
+
 # This is not a member of the ParallelGUFuncBuilder function because it is
 # called without an enclosing instance from parfors
 
-
-def build_gufunc_wrapper(py_func, cres, sin, sout, cache):
+def build_gufunc_wrapper(py_func, cres, sin, sout, cache, is_parfors):
```

Comment:
I think a comment on the effect of `is_parfors` would be useful.

Reply:
added comment in https://github.com/numba/numba/pull/4155/commits/37684eff506bdf7fbdaa00e373d06d9321d57409